.config/guardian/TSAConfig.gdntsa

```diff
@@ -0,0 +1,15 @@
+{
+    "codebaseName": "VSCode-CSharp-GitHub",
+    "notificationAliases": [
+        "mlinfraswat@microsoft.com"
+    ],
+    "codebaseAdmins": [
+        "REDMOND\\jaredpar",
+        "REDMOND\\arkalyan"
+    ],
+    "instanceUrl": "https://devdiv.visualstudio.com",
+    "projectName": "DevDiv",
```

Comment:
Is this correct?  Would have thought it would be dnceng

Reply:
Our issues are tracked in DevDiv